packer/config-drive.yml

```diff
@@ -13,7 +13,7 @@
       configdrive_fqdn: "test.example.com"
       configdrive_name: "openhpc2"
       configdrive_ssh_public_key: "{{ lookup('file', ssh_public_key_path) }}"
-      configdrive_config_dir: "/tmp/configdrive/"
+      configdrive_config_dir: "{{ playbook_dir }}"
```

Comment:
Worth adding the generated files to gitignore?

Reply:
They get deleted after a successful run anyway